### ISSUE TITLE
IP Oil conflicts with other mod's oils

### ISSUE THREAD

**Issue by justinrusso** [Flaxbeard/ImmersivePetroleum#93]
If mod that loads prior to Immersive Petroleum adds Oil, it causes conflicts. Recipes still appear to use Immersive petroleum, but the liquid is broken. The texture in the bucket is missing, etc.

#### Versions
**MC:** 1.12.2
**Forge:** 14.23.1.2599
**IP:** 1.1.6

**Comment by darthvader45** [comment]
I actually came here after noticing it.  Seems it needs a compat tweak in order to load it correctly.  Though IP oil may be a fluid, it does not generate physically.  Either way, it will need some tweaking in order to play nice with other mods that add oil (looking at you, Buildcraft).

**Comment by scottstone7** [comment]
been having an issue with this myself. i've be trying to set up a mod pack on a private server and keep running into issues with the crude oil. got all the ores sorted out but it seems to come back to the oil not being friendly.

**Comment by SSyl** [comment]
Just chiming in to say I was hoping to use Immersive Petroleum as pretty much the exclusive way to get oil for Galacticraft on 1.12 but this bug prevents that.

**Comment by Flaxbeard** [comment]
What error are you running into? Are you using the newest version?

**Comment by randomei** [comment]
@Flaxbeard I confirm this bug. I've installed galacticraft 4 after immersive petroleum on existing world and got the following error:
`[21:02:34] [Server thread/INFO]: Registry Block: Found a missing id from the world immersivepetroleum:fluid_crude_oil
[21:02:34] [Server thread/ERROR]: Unidentified mapping from registry minecraft:blocks
[21:02:34] [Server thread/ERROR]:     immersivepetroleum:fluid_crude_oil: 545`

And immersive petroleum oil now have no texture.

MC: 1.12.2
Forge: 14.23.2.2611
IP: 1.1.7

P.S. on new generated world I'm getting lava instead of oil, when using pumpjack

**Comment by Morpheus1101** [comment]
This is also occurring with BuildCraft

**Comment by darthvader45** [comment]
A commit was made, perhaps it will help with this issue.  Now to wait until a version is released with this commit in it.

**Comment by SSyl** [comment]
The problem seems to be fixed, at least when using Galacticraft. The pumpjack now automatically pumps in Galacticraft's Crude Oil (rather than IP's) when using Galacticraft's default settings.

**Comment by WhiteScorpion7** [comment]
I have the same issue with TechStack's Heavy Machinery Mod. Oil have not texture and not transfuse from barrels to buckets.

**Comment by Freebi85** [comment]
Same Here :(

**Comment by Flaxbeard** [comment]
Is this still an issue with 1.1.9?

**Comment by fira** [comment]
@Flaxbeard Yup. Afraid so. Tried to launch with GC, IP and BC on at same time, and issues occurs during texture stitching. They don't play nice between each other. I believe GC has a specific compatibility patch if it detects Buildcraft, but not for IP, and IP would have this problem with Buildcraft on his own aswell
